### PR TITLE
Homework has been done

### PR DESCRIPTION
**Pull request by alqaesi**

### CODE REVIEW

**Review by Pshenichnyi-Aleksey — APPROVED** [review]

**Review by Sarmat92 — APPROVED** [review]
fmig

File: homework/english/alqaesi/lecture 1/task1.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+#GRAMMAR PRACTICE
+
+TASK1
+
+1 He usually has breakfast at 6 a.m.
+2 Anyone who smokes 20 cigarettes a day can expect to have health problems.
+3 She doesn't like cooking, but today she bakes a pie.
```
Comment (Uchukaknado) [inline homework/english/alqaesi/lecture 1/task1.md:7]: She doesn't like cooking, but today she's baking a pie.

File: homework/english/alqaesi/lecture 1/task1.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+4 I am finishing the project right now.
+5 I don't listen to rock music very often.
+6  — What does your sister do?
+— She is a programmer.
+
+TASK2
+
+1 Do you usually work on Saturdays?
+2 Listen! Somebody singing.
```
Comment (Uchukaknado) [inline homework/english/alqaesi/lecture 1/task1.md:16]: Listen! Somebody is singing.

File: homework/english/alqaesi/lecture 1/task1.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+3 Look! The sun is shining.
+4 He doesn't use his car every day.
+5 — Where do you come from? ( Where are you from)
```
Comment (Uchukaknado) [inline homework/english/alqaesi/lecture 1/task1.md:19]: Both options are okay!

**Review by Uchukaknado — COMMENTED** [review]
Well done! Try to come up with your own examples next time ;) in this way you'll get more from the tasks.

**Review by VasiliZas — APPROVED** [review]

**Pull request merged**